### ISSUE TITLE
[3.8] Class Aliasing Issue

### ISSUE THREAD

**Issue by mbabker** [joomla/joomla-cms#16311]
### Steps to reproduce the issue

Enable debug plugin's logging

### Expected result

Works

### Actual result

```sh
Argument 1 passed to PlgSystemDebug::logger() must be an instance of JLogEntry, instance of Joomla\CMS\Log\LogEntry given
```

**Comment by laoneo** [comment]
#16342 workarounds the issue.

**Comment by brianteeman** [comment]
closed by #16342
